_candidates_en/SotaroKimura_1.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Sotaro Kimura
+title: "Start playing with Spark Structured Streaming locally and build up a stream processing pipeline"
+length: 40
+audience: Beginner
+language: Japanese
+twitter: kimutansk
+github: kimutansk
+icon: https://s.gravatar.com/avatar/5d869b93035410ece41ff0d86d8c5c14
+organization: DWANGO Co., Ltd.
+tags:
+  - Big Data / Fast Data
+  - Tools
+suggestions:
+  - People who are interested in Spark and want to learn basics
+  - People who want to know practical use cases of Spark
+  - People who want to build a stream processing system quickly
+---
+Spark is a framework for parallel stream processing, however, that probably sounds like something difficult to work with even if you are interested.
+Actually, Spark is a data processing tool which allows you execute it on your local machine as well as a large-scale cluster.
```

Comment:
Consider changing to "+In reality, Spark is a data processing tool which allows you to execute it on your local machine as well as in a large-scale cluster."